spec/controllers/application_controller_spec.rb

```diff
@@ -196,4 +196,65 @@
       end
     end
   end
+
+  describe "#build_user_emails_for_edit" do
+    before :each do
+      EvmSpecHelper.local_miq_server
+      MiqUserRole.seed
```

Comment:
Thanks, these tests are much easier to read.  Do we need to create all of the roles?  It looks like we only use EvmRole-operator, does this test pass if we only create that one user role?